src/main/java/org/jabref/gui/bibtexkeypattern/BibtexKeyPatternPanel.java

```diff
@@ -97,6 +115,11 @@ private void buildGUI() {
         JButton btnDefault = new JButton(Localization.lang("Default"));
```

Comment:
old jbutton. Please remove all rest of the old swing stuff and it would be good to merge